api/v1/swagger.yaml

```diff
@@ -3829,6 +3829,9 @@ paths:
               context:
                 type: object
                 description: Any object, typically JSON, used to describe to some infomration that needed to be logged.
+              type:
```

Comment:
what about adding type to the list of attributes you can filter from query params, e.g. ...&type=foo

Reply:
Thanks Ian, good catch, done